src/lib/storage/proposal/partition_creator.rb

```diff
@@ -350,7 +345,10 @@ def create_physical_volumes(volume_group)
         # @param strategy     [Symbol] :desired or :min_size
         #
         def create_logical_volume(volume_group, vol, strategy)
-          log.info("Creating LVM logical volume #{vol.logical_volume_name} with strategy \"#{strategy}\"")
+          log.info(
+            "Creating LVM logical volume #{vol.logical_volume_name} at #{volume_group} "\
+            "with strategy \"#{strategy}\""
+          )
```

Comment:
Well, in code review and in manual testing. The proposal behavior is exactly the same in two systems before and after the changes.